backend/tests/unit/test_config.py

```
@@ -0,0 +1,171 @@
+            api_key=SecretStr("custom-key"),  # SecretStrでラップして渡す
+            log_level="DEBUG",
+        )
+
+        # Assert: カスタム値が正しく設定されることを検証
+        assert config.host == "0.0.0.0"
+        assert config.port == 9000
+        assert config.reload is False
+        assert config.api_key.get_secret_value() == "custom-key"
+        assert config.log_level == "DEBUG"
+
+    def test_from_env_missing_r2_raises_error(self, monkeypatch):
```

Ruff (0.14.10)

Unused method argument: `monkeypatch`

(ARG002)